climada/entity/exposures/crop_production.py

```diff
@@ -37,27 +37,32 @@
 logging.root.setLevel(logging.DEBUG)
 LOGGER = logging.getLogger(__name__)
 
-DEF_HAZ_TYPE = 'CP'
+DEF_HAZ_TYPE = 'RC'
 """Default hazard type used in impact functions id."""
 
 BBOX = np.array([-180, -85, 180, 85])  # [Lon min, lat min, lon max, lat max]
+"""Default geographical bounding box"""
```

Comment:
Why these values? Comment should explain.

Reply:
Was this point addressed? What do the default numbers refer to? Is this the whole world? The global South? The global North?